lib/msal-core/test/Utils.spec.ts

```diff
@@ -14,4 +15,22 @@ describe("Utils.ts class", () => {
         console.log(Utils.replaceTenantPath("http://a.com/common/", "1234-56778"));
         console.log(Utils.replaceTenantPath("http://a.com/common", "1234-5678"));
     });
+
+    it("test Base64 encode decode", () => {
```

Comment:
I think the right thing to test here would be `Utils.base64EncodeStringUrlSafe`  instead of `Base64.encode`

Reply:
Well, this was intentional. Those tests are coming as a part of Utils.ts testing, base64DecodeStringUrlSafe is used for idToken and no one calls base64EncodeStringUrlSafe as far as I can see. I wanted to construct more meaningful inputs for these APIs that are realistic as a part of the Utils.ts UT suite, hence was testing only Base64.encode() and Base64.decode() in this specific case.

Reply:
I would say us testing another libs api is less useful than testing our own. We did talk offline, but if api is unused, we can delete. otherwise, if we leave it in, lets test that interface instead of Base64.<>

Reply:
Done, please review. Code removal will be done later in a patch, raised a VSTS for the same.